wyatt_test.go

```diff
@@ -83,6 +86,26 @@ func TestUnmarshal(t *testing.T) {
 
 		assert.EqualValues(t, expectedOutuput, actualOutput)
 	})
+
+	t.Run("ignores environment variables not prefixed with 'INPUT_'", func(t *testing.T) {
+		type simpleInputStruct struct {
+			StringInput   string `json:"string"`
+			AnotherString string `json:"another_string"`
+		}
+
+		t.Setenv("INPUT_STRING", "a string")
+		t.Setenv("ANOTHER_STRING", "another string")
```

Comment:
I believe `t.Setenv` uses `os.Setenv` under the hood, but also restores environment variables to their original values during cleanup.